test/verify_signature.test.ts

```diff
@@ -0,0 +1,146 @@
+import chai, { expect } from "chai";
+import hre, { ethers, starknet } from "hardhat";
+import { StarknetContract, HardhatUserConfig } from "hardhat/types";
+import config from "../hardhat.config";
+
+const ASSET_SYMBOL = 10703902247957299200n;
```

Comment:
detail: most values here are also in the other test file. you could create a file `constants.ts` in your `test` repo where you would move these values.